### PR TITLE
Add Viability Testing empty state

### PR DESCRIPTION
**Pull request by constanzauanini**
Screen Shot 2022-09-15 at 16 13 27

### CODE REVIEW

File: src/components/accession2/NewTestModal.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import strings from 'src/strings';
+
+export interface NewTestModalProps {
+  open: boolean;
+  accession: Accession2;
+  onClose: () => void;
+  reload: () => void;
+}
+
+export default function NewTestModal(props: NewTestModalProps): JSX.Element {
```
Comment (karthikbtf) [inline src/components/accession2/NewTestModal.tsx:13]: `NewViabilityTestModal`? suggestion for file name and class name and where it's used.

**Review by karthikbtf — APPROVED** [review]
A suggestion for file/class name but up to you. Feel free to merge.

**Review by karthikbtf — APPROVED** [review]
Thanks 🎉
